includes/definitions/discovery/procurve.yaml

```diff
@@ -144,3 +150,20 @@ modules:
                         - { descr: unknownState , graph: 1, value: 10, generic: 3 }
                         - { descr: standbyBooting , graph: 1, value: 11, generic: 3 }
+                -
+                    oid: entStateOper
+                    value: entStateOper
+                    num_oid: '.1.3.6.1.2.1.131.1.1.1.3.{{ $index }}'
+                    descr: '{{ $entPhysicalName }} [{{ $entPhysicalDescr }}]'
+                    index: 'entStateOper.{{ $index }}'
+                    state_name: entStateOper
+                    states:
+                        - { descr: notPresent, graph: 1, value: 1, generic: 3 }
+                        - { descr: unknown, graph: 1, value: 2, generic: 1 }
+                        - { descr: ok, graph: 1, value: 3, generic: 0 }
+                        - { descr: failed, graph: 1, value: 4, generic: 2 }
+                    skip_values:
+                        -
+                            oid: entPhysicalDescr
+                            op: 'in_array'
+                            value: ['', 'Chassis Sensor', 'Aruba JL073A Switch interface module pseudocontainer', 'Aruba JL073A Switch power supply bay pseudocontainer', 'Aruba JL073A Switch temperature sensor', 'Aruba JL073A Switch Management Module Slot', 'Aruba JL073A Switch port module slot', 'Aruba JL073A Switch power supply bay', 'Aruba JL073A Switch power supply', 'Aruba JL073A Switch fan', 'Aruba JL073A Switch Stacking Port', 'Aruba JL073x Fixed 24p 1GBase-T PoE+ Module', 'Aruba JL073x No Card', 'Aruba JL073x 100/1000BASE-T Port', 'Aruba JL074A Switch interface module pseudocontainer', 'Aruba JL074A Switch power supply bay pseudocontainer', 'Aruba JL074A Switch temperature sensor', 'Aruba JL074A Switch Management Module Slot', 'Aruba JL074A Switch port module slot', 'Aruba JL074A Switch power supply bay', 'Aruba JL074A Switch power supply', 'Aruba JL074A Switch fan', 'Aruba JL074A Switch Stacking Port', 'Aruba JL074x Fixed 24p 1GBase-T PoE+ Module', 'Aruba JL074y Fixed 24p 1GBase-T PoE+ Module', 'Aruba JL074x No Card', 'Aruba JL074y No Card', 'Aruba JL074x 100/1000BASE-T Port', 'Aruba JL074y 100/1000BASE-T Port', 'Aruba JL083A 4-port Stacking Module', 'Aruba JL084A 4-port Stacking Module', 'HP J9850A Switch interface module pseudocontainer', 'HP J9850A Switch power supply bay pseudocontainer', 'HP J9850A Switch temperature sensor', 'HP J9850A Switch Management Module Slot', 'HP J9850A Switch port module slot', 'HP J9850A Switch power supply bay', 'HP J9850A Switch power supply', 'HP J9850A Switch fan', 'HP J9851A Switch interface module pseudocontainer', 'HP J9851A Switch power supply bay pseudocontainer', 'HP J9851A Switch temperature sensor', 'HP J9851A Switch Management Module Slot', 'HP J9851A Switch port module slot', 'HP J9851A Switch power supply bay', 'HP J9851A Switch power supply', 'HP J9851A Switch fan', 'HP J9990A 100/1000BASE-T Port', 'HP J9990A 10GIGBASE-GEN Port', 'HP J9990A No Card']
```

Comment:
This feels like a personal preference on what you'd not want to see, so I'm not sure we could accept it.

However, this change seems to get you what you need and feels more maintainable.

```
                    skip_values:
                        -
                            oid: entPhysicalModelName
                            op: '='
                            value: ''
```

And nets you the below from your test data (one more sensor than before).
<img width="845" alt="image" src="https://github.com/user-attachments/assets/7b95bc8f-b981-415a-8b14-f2020c6b980a" />

Reply:
Oh and don't forget to add `entPhysicalModelName` to pre-cache.

Reply:
@laf - It was almost a perfect solution, but using 'entPhysicalModelName' removes the 'Switch Management Module' value, which is a critical component to monitor in Procurve switches. Is there any way we can get that value back? If so, your solution is spot on.

Reply:
Have a play around with the data you have, this might get you closer

```
                    skip_values:
                        -
                            oid: entPhysicalClass
                            op: 'not_in_array'
                            value: [3,4,5]
```

<img width="864" alt="image" src="https://github.com/user-attachments/assets/e127c6a3-2e99-4826-b976-80ed6529338d" />